lepton/package.go

```diff
@@ -373,19 +377,23 @@ func ExtractPackage(archive, dest string, config *types.Config) {
 		case tar.TypeDir:
 			if _, err := os.Stat(target); err != nil {
 				if err := os.MkdirAll(target, 0755); err != nil {
-					panic(err)
+					fmt.Printf("Failed to create directory %s", err)
```

Comment:
It would be good to print the directory path as well, i.e. `target`

Reply:
done